spring-ai-core/src/main/java/org/springframework/ai/aot/SpringAiCoreRuntimeHints.java

```diff
@@ -43,7 +43,7 @@ public class SpringAiCoreRuntimeHints implements RuntimeHintsRegistrar {
 	public void registerHints(@NonNull RuntimeHints hints, @Nullable ClassLoader classLoader) {
 
 		var chatTypes = Set.of(AbstractMessage.class, AssistantMessage.class, ToolResponseMessage.class, Message.class,
-				MessageType.class, UserMessage.class, SystemMessage.class, FunctionCallbackContext.class,
+				MessageType.class, UserMessage.class, SystemMessage.class, DefaultFunctionCallbackResolver.class,
```

Comment:
How does this work when we have other implementations of "FunctionCallbackResolver"

Reply:
If there are other implementation they have to be added here.